tests/unit/plugins/cache/test_redis.py

```diff
@@ -19,14 +19,31 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
+import os
```

Comment:
What's this needed for?

Reply:
Oh, leftover from something I was testing. Removed.